geometry_msgs/PoseStamped target_pose
string object_name
---
string result
geometry_msgs/PoseStamped updated_goal

---
geometry_msgs/PoseStamped current_pose
string feedback
